.github/workflows/ci.yml

```
@@ -27,6 +27,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: pnpm/action-setup@v2.2.4
+        with:
+          version: ${{ matrix.node-version == 20 && 8 || 7 }}
```

it's a pretty uncommon way of writing this. might be clearer with a ternary. also perhaps we should use pnpm 8 everywhere we can if we're going to be using multiple versions:
```suggestion
          version: ${{ matrix.node-version !== 14 ? 8 : 7 }}
```

---

Does this support a ternary? I don't know how these template expressions are processed. (I would also bet they don't support `!==` anyway, which is definitely a JS-ism.)

---

> Does this support a ternary

AFAIK no. That's why I choose this way.
https://github.com/actions/runner/issues/409#issuecomment-727565588

I have no strong opinion that we will mainly use pnpm7 or pnpm8.

---

Ah, nonetheless, I think my preference would probably be to just drop Node 14 and use pnpm 8 everywhere. What do you guys think of that option?

---

For Svelte 5 we can do this, but for Svelte 4 I think it's too soon

---

It looks like the new suggestion would use pnpm 7 on _not_ 14, and use 8 on 14. `!=` should be `==`.

---

yes, thank you! updated